Sources/OTelSpanConventions/Generated/SpanAttributes+http.swift

```diff
@@ -0,0 +1,268 @@
+// DO NOT EDIT. This file is generated automatically. See README for details.
+
+// swiftlint:disable all
+
+import OTelConventions
+import Tracing
+
+extension SpanAttributes {
+    /// `http` namespace
+    public var http: HttpAttributes {
+        get {
+            .init(attributes: self)
+        }
+        set {
+            self = newValue.attributes
+        }
+    }
+    
+    @dynamicMemberLookup
+    public struct HttpAttributes: SpanAttributeNamespace {
+        public var attributes: SpanAttributes
+    
+        public init(attributes: SpanAttributes) {
+            self.attributes = attributes
+        }
+    
+        public struct NestedSpanAttributes: NestedSpanAttributesProtocol {
+            public init() {}
+            /// `http.route`: The matched route, that is, the path template in the format used by the respective server framework. 
+            /// 
+            /// - Stability: stable
+            /// 
+            /// - Type: string
```

Comment:
Do we want to try to get rid of the type documentation here since it's already translated into the Swift type system anyways?

Reply:
We definitely could. I figured that including the full OTel docs might be helpful for clarity's sake, since they don't always match up quite this cleanly between OTel and Swift ([templateStringArray example](https://github.com/NeedleInAJayStack/swift-otel-semantic-conventions/blob/feat/generator/Sources/OTelConventions/Generated/OTelAttribute+http.swift#L27)). It's not too much effort to remove the OTel type in spots where we've already translated to Swift, though. Let me know if you'd like it removed, and I'd be happy to.

Reply:
Yeah, that's a good point. I'm happy to keep it this way then.